changelog/README.rst

```diff
@@ -0,0 +1,30 @@
+This directory contains "newsfragments" which are short that contain a small **ReST**-formatted
+text that will be added to the next ``CHANGELOG``.
+
+The ``CHANGELOG`` will be read by users, so this description should be aimed to pytest users
+instead of describing internal changes which are only relevant to the developers.
+
+Make sure to use full sentences with correct case and punctuation, for example: *Fix issue with non-ascii contents in doctest text files.*
+
+Each file should be named like ``<ISSUE>.<TYPE>.rst``, where
+``<ISSUE>`` is an issue number, and ``<TYPE>`` is one of:
+
+* ``feature``: new user facing features, like new command-line options and new behavior.
+* ``bugfix``: fixes a reported bug.
+* ``doc``: documentation improvement, like rewording an entire session or adding missing docs.
+* ``removal``: feature deprecation or removal;
+* ``vendor``: vendoring news;
```

Comment:
Might want to be a bit verbose here and give some context on what vendoring means - something like "Changes in packages vendored in pytest." or so? On the other hand, didn't we get rid of vendoring anyways?

Reply:
Good point, updated.

We are not vendoring anything more at the moment, but we might do it again in the future. I guess we can always remove this later if we officially abolish the practice.